src/Components/Component1/Constants.js

```diff
@@ -0,0 +1,86 @@
+export default DUMMY_JSON_PAYLOAD = [
```

Comment:
When you ```export default``` something, you should not name it somehow - [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)